modules/ossm-cr-status.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-status_{context}"]
+= Service Mesh status
+
+The information in your `ServiceMeshControlPlane` file under `status` describes the current state of your service mesh.  This information is generated by the Operator and is read-only.
```

Comment:
Is there not an attribute for Service Mesh to keep the capitalization consistent? I'd update the title here maybe if it's not referring to a product name or something to "Service mesh status"

Reply:
Is status a spec parameter as well? I'd maybe phrase this like the others in that case to make it consistent, e.g.

"The `spec.status` parameter describes the current state of your service mesh. This information is generated by the Operator and is read-only."